src/modules/taskHelper.js

```diff
@@ -0,0 +1,53 @@
+class TaskHelpers {
+  static CreateBookEntry(existingTasks, taskDescription) {
+    const id = existingTasks.length === 0 ? 0 : existingTasks.length;
+    const updatedTasks = existingTasks;
+
+    updatedTasks.push({
+      index: id,
+      description: taskDescription,
+      completed: false,
+      cleared: false,
+    });
+
+    return updatedTasks;
+  }
+
+  static UpdateTaskStore = (tasks) => {
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  };
```

Comment:
- [ ] Great job so far 👍 but when adding tasks, the first task should have an `index` of 1 but your index is starting from 0. Kindly address this issue.
- Local Storage:
![{4F156C3B-C48E-46CB-B28E-41D492C0A039} png](https://user-images.githubusercontent.com/85113567/154828640-98d2ce53-1973-46c9-a456-0ac6f2ea8e1d.jpg)